flex-cn-setup/pygluu/kubernetes/templates/helm/gluu/charts/opendj/templates/statefulset.yaml

```diff
@@ -29,45 +29,8 @@ spec:
       {{- end }}
     spec:
       securityContext:
-        runAsUser: 1000
-        runAsNonRoot: true
-      initContainers:
-      - command:
-        - /bin/sh
-        - -c
-        - ls -l /opt/opendj; /bin/chown -R 1000:1000 /opt/opendj || echo "did not work"; ls -l /opt/opendj; sleep 5
-        image: busybox:latest
-        imagePullPolicy: Always
-        name: change-folders-ownership
-        resources: {}
-        terminationMessagePath: /dev/termination-log
-        terminationMessagePolicy: File
-        volumeMounts:
-        {{- range $key, $values := $.Values.openDjVolumeMounts }}
-          - mountPath: {{$values.mountPath}}
-            name: {{$values.name}}
-            subPath: {{$key}}
-        {{- end }}
-        {{- with $.Values.volumeMounts }}
-{{- toYaml . | nindent 10 }}
-        {{- end }}
-    {{- with $.Values.image.pullSecrets }}
-      imagePullSecrets:
-        {{- toYaml . | nindent 8 }}
-    {{- end }}
-      dnsPolicy: {{ $.Values.dnsPolicy | quote }}
-    {{- with $.Values.dnsConfig }}
-      dnsConfig:
-{{ toYaml . | indent 8 }}
-    {{- end }}
-      volumes:
-      {{- with $.Values.volumes }}
-{{- toYaml . | nindent 8 }}
-      {{- end }}
-      {{- if .Values.topologySpreadConstraints }}
-      topologySpreadConstraints:
-        {{- include "opendj.topology-spread-constraints" . | indent 8 }}
-      {{- end }}
+        runAsUser: 0
```

Comment:
The running user must not root user to avoid security issue.

We used initContainer to change ownership of mounted volumes in fresh and existing deployment.